lib/src/features/ipc/handlers/get_trades_handler.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:pylons_sdk/src/core/constants/strings.dart';
+import 'package:pylons_sdk/src/features/ipc/base/ipc_handler.dart';
+import 'package:pylons_sdk/src/features/ipc/responseCompleters.dart';
+import 'package:pylons_sdk/src/features/models/sdk_ipc_response.dart';
+
+import '../../../../pylons_sdk.dart';
+
+class GetTradesHandler implements IPCHandler {
+  @override
+  void handler(SDKIPCResponse<dynamic> response) {
+    final defaultResponse = SDKIPCResponse<List<Trade>>(
+        success: response.success,
+        action: response.action,
+        data: [],
+        error: response.error,
+        errorCode: response.errorCode);
+    try {
+      if (response.success) {
+        defaultResponse.data = List.from(response.data).map((e) {
+          return Trade.create()..mergeFromProto3Json(e);
+        }).toList();
+      }
+    } on Exception catch (_) {
+      defaultResponse.error = 'Recipe parsing failed';
```

Comment:
Its not a recipe its a trade.

Reply:
Thanks for pointing this out. It's fixed.